skyportal/models.py

```diff
@@ -405,6 +405,8 @@ class Photometry(Base):
     altdata = sa.Column(JSONB)
     upload_id = sa.Column(sa.String, nullable=False,
                           default=lambda: str(uuid.uuid4()))
+    alert_id = sa.Column(sa.BigInteger, nullable=False, unique=True,
```

Comment:
```suggestion
    alert_id = sa.Column(sa.BigInteger, nullable=True, unique=True,
```

Not all photometry comes from alerts, this should be nullable to accommodate e.g., follow up photometry